resources/assets/components/InboxItem/index.js

```diff
@@ -52,12 +70,12 @@ class InboxItem extends React.Component {
           </ul>
         </div>
         <div className="container__block -third">
-          {post['user'] ?
+          {post['user'] || signup.user ?
             <div>
-              <h2>{post['user']['first_name']} {post['user']['last_name']}, {calculateAge(post['user']['birthdate'])}</h2>
+              <h2>{first_name} {last_name}, {birthdate}</h2>
```

Comment:
It might make sense to use the Northstar user's [`displayName`](https://github.com/DoSomething/gateway/blob/30565f68e35fe1051be0569c8292e643f9a8c499/src/Resources/NorthstarUser.php#L21-L36) here!